Source/WebCore/platform/graphics/gstreamer/WebKitWebSourceGStreamer.cpp

```diff
@@ -976,9 +976,9 @@ void CachedResourceStreamingClient::dataReceived(PlatformMediaResource&, const c
     // to push buffers that are too large, otherwise incorrect buffering messages can be sent from the
     // pipeline.
     uint64_t bufferSize = gst_buffer_get_size(priv->buffer.get());
```

Comment:
This does not belong to this patch so I guess we can forget, but just a quick heads up: we don't need to call `gst_buffer_get_size` because unless I am missing something in lines 959-962, it is ensured that `bufferSize == length`, sign apart...

Reply:
That's true, it's probably clearer to call the size API, but can be cleaned up.